script.js

```diff
@@ -1,12 +1,210 @@
+let _localEpisodes = [];
+let _localShows = [];
+let _showDDLCtrlId = "selShow";
+let _epDDLCtrlId = "selName";
```

Comment:
I really like your top 2 variables, but the bottom 2 might be confusing to people who are unfamiliar with your code, it's a good idea to make your variable names as explicit as possible